### PR TITLE
Internal: Restrict users from creating new Theme Builder parts when license is inactive [HTS-22]

### PR DESCRIPTION
**Pull request by mark-elementor**
## PR Checklist

- [x] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: app/services/account/account-service.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+export default class AccountService {
+	/**
+	 * Authorisation service.
+	 *
+	 * @param {string} buttonRef
+	 * @param {string} parseUrl
+	 * @param {Array} sizes
+	 * @since 3.9.0
```
Comment (ManorHazaz) [inline app/services/account/account-service.js:8]:
```suggestion
	 * @since 3.10.0
```

File: app/services/base-service.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?php
+namespace Elementor\App\Services;
+
+abstract class Base_Service {
+	/**
```
Comment (ManorHazaz) [inline app/services/base-service.php:5]:
```suggestion

	/**
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — DISMISSED** [review]

**Review by nivnoimanelementor — DISMISSED** [review]
Looks good, just update branch please.

**Review by rodolphebertozzo — COMMENTED** [review]
Please, see: https://github.com/mark-elementor/elementor/pull/1

**Pull request closed without merging**